### ISSUE TITLE
Some discovery after installing windows11

### ISSUE THREAD

**Issue by JBKing514** [olm3ca/Galaxy-Chromebook#11]
1. The Samsung dedicated UFS card was confirmed to work on both windows11/Manjaro with seq read around 372MB/s and seq write around 202MB/s. And the uSD card works as well.
![UJ6HK$SR@VG25H_ @L%E3HI](https://user-images.githubusercontent.com/65148386/199938687-ca7d84fe-ab77-4ba6-951c-d5788254879d.png)
2. It Seems the type c video output didn't work on either windows or Manjaro(despite the sound working), now using the USB A display link adapter as an alternative.
3. Elan Touchscreen seems working after installing the updated crostouchscreen.2.9.3 driver.
![`%U1TZ%S80~K_PERDU4A{3E](https://user-images.githubusercontent.com/65148386/199940040-4f8c9e78-2c0b-45cf-aaab-a98ea0e73e41.png)
4. Driver download links in README were no longer available but can still be downloaded on coolstar's website
5. I disabled my CR50 by disconnecting the battery because I don't have and don't want to buy a Suzy cable, maybe add that in README. 
6. Both cameras were working on windows11 but had no mic because of a lack of drivers.
7. Keyboard backlit is working on both windows11/Manjaro
8.  S pen and its pressure sense works by default without any drivers
9. Filping the laptop 360 can automatically activate the tablet mode in windows11

That's all discovery I found after trying custom UEFI firmware, hope it helps.

**Comment by olm3ca** [comment]
Wow, great post. Thanks so much. I'll leave this issue open because I want people to see your findings. Thank you - I updated the README a bit with your info.

**Comment by vvvsrx** [comment]
感谢楼主的信息!这就去装Win11!

**Comment by JBKing514** [comment]
Update:
Recently I installed Ubuntu 22.04.1 LTS and found it seems working better than Manjaro because not only its sound working out of the box, but the sound level adjustment also works fine. In Manjaro the sound has only an on/off state and the sound level can't be adjusted. And I can finally able to use the right USB C port(the left port can't)  with a USB C-HDMI adapter to connect to other displays. (still can't do that in Windows11)

![20221201_185725](https://user-images.githubusercontent.com/65148386/205023626-c2b769dc-4426-4b20-8782-b7b5981ff0dc.jpg)

I'm currently running Ubuntu on my UFS card and it feels fly. The extra performance compares to traditional TF cards is good enough for Linux to run very fluently.

**Comment by hung0702** [comment]
Can you comment about battery life under Linux/Windows? More interested in Windows.

Anyone willing to try ThrottleStop? I'm trying to pick up a Galaxy Chromebook since the progress has gotten mature.

**Comment by JBKing514** [comment]
> For my current experience, in Windows 11 I can get about 5-6 hours of screen on time with some light text editing and web browsing which is almost the same with ChromeOS. And ubuntu has the best battery life about 6-7 hours. But I strongly recommend you don't buy this laptop because it has severe touchpad issues which make the touchpad almost unusable. You can check the comment on Samsung's official page about this laptop, most buyers had the same touchpad issue including me which is very frustrating.

**Comment by hung0702** [comment]
Have you considered the [copper tape fix](https://m.imgur.com/a/GDZayv9) ? Many people think the issue is due to inadequate grounding, and simply adding two little strips of copper (or other conductive metal) tape should resolve the issue.

**Comment by JBKing514** [comment]
> Have you considered the [copper tape fix](https://m.imgur.com/a/GDZayv9) ? Many people think the issue is due to inadequate grounding, and simply adding two little strips of copper (or other conductive metal) tape should resolve the issue.

Thanks so much for your info. I will try to fix that, fingers crossed.

**Comment by JBKing514** [comment]
![)75875 7ER2TL7%6 L`3B8J](https://user-images.githubusercontent.com/65148386/214214623-5fb629a0-a97b-4394-82d5-09a4a71cf033.png)
A new audio driver from Coolstar had been released. It supports Gemini Lake, Comet Lake(including Kohaku), Jasper Lake, Tiger Lake, and Alder Lake CPU. I bought the driver and confirmed that the speaker, mic, and headphone jack were all functional, cheers.

**Comment by olm3ca** [comment]
Yes, really great news for kohaku owners. Congrats! Added updates to the readme.

**Comment by hung0702** [comment]
Unfortunately I'm not able to license the audio drivers due to my SSD not showing its serialnumber in Windows? Can you try:

Windows `wmic` `diskdrive get name,model,serialnumber`  (I got `FFFF_FFFF_FFFF_FFFF.`)

Linux live USB, `nvme-cli` `nvme list` returns a <17char> serial of random numbers/letters.

I returned to stock using MrChromebox's script. In ChromeOS I went to set the HWID but saw it was already KOHAKU-JPZQ <15char> and proceeded to install full UEFI > shutdown.

Insert USB > install Win11 > install chipset/SATA drivers. I checked SDIO and updated Intel IO/Chipset/DPTF/SATA drivers to the highest version (or most recent) unsigned drivers available and rebooted a couple times. The wmic commands still return FFFF while the nvme-cli returns the same 17char.

**Comment by JBKing514** [comment]
Update about installing Brunch on UFS card@4/18/2023
Although I don't think anyone will do the same thing as trying to install Brunch on the UFS card, I still want to leave a comment here just for reference.

If you only want to know how to install Brunch on the UFS card, please skip to the solution part.

Problem:
Recently I was trying to install Brunch on my UFS card because my main nvme drive was already loaded with Windows 11 and all the drivers. So the option left is to install the Brunch into my UFS card, and as you can see from the top of this issue, it has SSD-like random IO performance which makes it suitable to install OS on that. 

I followed the guide about installing Brunch on GitHub [https://github.com/sebanc/brunch/blob/main/install-with-windows.md](url) and created an installing image. But when I tried to burn the .img file to the UFS card using Rufus, I was been warned that the UFS card uses 4096 bytes as sector size rather than the traditional 512 bytes and it will unable to create a bootable drive on the UFS card. I tried other burning tools but none of them worked. I also tried to install the OS by following the single boot guide but the only thing I got is a broken ESP(boot) partition. Then I tried to follow the dual boot guide for Windows using grub2win but I ended up with a restart loop.

Solution:
Then I thought before I installed Ubuntu on my UFS card and its grub works just fine, so maybe the dual boot for Linux can help me. And as I expected, it worked. First I install Ubuntu on my UFS card using the iso file normally. Then I adjust the data partition of that Ubuntu installment using the disk utility in live CD so I can have a free space at the end for installing Brunch. Finally, by following the guide of Linux dual boot [https://github.com/sebanc/brunch/blob/main/install-with-linux.md](url), and updating the grub menu, I was able to boot Brunch successfully using grub created by Ubuntu and it solves the problem. Now I have a triple boot which includes Windows 11 on nvme, and Ubuntu and Brunch on the UFS card.
![20230417_231126](https://user-images.githubusercontent.com/65148386/232647671-21b45386-34e7-4881-8bae-f2a42e6120d4.jpg)
![20230417_231012](https://user-images.githubusercontent.com/65148386/232647710-0c7967be-71d4-4d7b-8068-4e4aa7481784.jpg)

**Comment by hung0702** [comment]
I've been looking for an easier guide to dual-boot ChromeOS Flex. Man, forget that, I'll just use brunch and get Android apps too. What ChromeOS version does it run?

**Comment by JBKing514** [comment]
> I've been looking for an easier guide to dual-boot ChromeOS Flex. Man, forget that, I'll just use brunch and get Android apps too. What ChromeOS version does it run?

The latest recovery image you can download is ver.111 and that is also the version that I am currently running. But I belive you can upgrade that by change some settings in Brunch, just follow the instruction in the README of this project to change those settings.

**Comment by olm3ca** [comment]
Yes, just make sure you have the line in grub that enables updates - for example `options=native_chromebook_image,enable_updates` 

One thing to note though is that brunch isn't always compatible with the latest ChromeOS. Sometimes you're better off staying on a stable release for a while.

**Comment by VariousTrick** [comment]
does elan touchscreen fix?
elan触摸屏可用了吗？
